sql/telemetry_derived/devtools_accessiblility_panel_usage_v1/query.sql

```diff
@@ -0,0 +1,134 @@
+    udf.get_key(
+      payload.processes.parent.keyed_scalars.devtools_accessibility_select_accessible_for_node,
+      "browser-context-menu"
+    ) AS browser_context_count,
+    udf.get_key(
+      payload.processes.parent.keyed_scalars.devtools_accessibility_select_accessible_for_node,
+      "inspector-context-menu"
+    ) AS inspector_context_count,
+    udf.get_key(
+      payload.processes.parent.keyed_scalars.devtools_accessibility_audit_activated,
+      "ALL"
+    ) AS audit_all_count,
+    udf.get_key(
+      payload.processes.parent.keyed_scalars.devtools_accessibility_audit_activated,
+      "CONTRAST"
+    ) AS audit_contrast_count,
+    udf.get_key(
+      payload.processes.parent.keyed_scalars.devtools_accessibility_audit_activated,
+      "KEYBOARD"
+    ) AS audit_keyboard_count,
+    udf.get_key(
+      payload.processes.parent.keyed_scalars.devtools_accessibility_audit_activated,
+      "TEXT_LABEL"
+    ) AS audit_text_label_count,
+    udf.get_key(
+      payload.processes.parent.keyed_scalars.devtools_accessibility_accessible_context_menu_item_activated,
+      "print-to-json"
+    ) AS print_to_json_context_count,
+    udf.get_key(
+      payload.processes.parent.keyed_scalars.devtools_accessibility_simulation_activated,
+      "accessibility.simulation.deuteranomaly"
+    ) AS simulation_deuteranomaly_count,
+    udf.get_key(
+      payload.processes.parent.keyed_scalars.devtools_accessibility_simulation_activated,
+      "accessibility.simulation.protanomaly"
+    ) AS simulation_protanomaly_count,
+    udf.get_key(
+      payload.processes.parent.keyed_scalars.devtools_accessibility_simulation_activated,
+      "accessibility.simulation.protanopia"
+    ) AS simulation_protanopia_count,
+    udf.get_key(
+      payload.processes.parent.keyed_scalars.devtools_accessibility_simulation_activated,
+      "accessibility.simulation.deuteranopia"
+    ) AS simulation_deuteranopia_count,
+    udf.get_key(
+      payload.processes.parent.keyed_scalars.devtools_accessibility_simulation_activated,
+      "accessibility.simulation.tritanopia"
+    ) AS simulation_tritanopia_count,
+    udf.get_key(
+      payload.processes.parent.keyed_scalars.devtools_accessibility_simulation_activated,
+      "accessibility.simulation.tritanomaly"
+    ) AS simulation_tritanomaly_count,
+    udf.get_key(
+      payload.processes.parent.keyed_scalars.devtools_accessibility_simulation_activated,
+      "accessibility.simulation.contrastLoss"
+    ) AS simulation_contrast_loss_count
+  FROM
+    telemetry.main
+  WHERE
+    sample_id = 1
+    AND DATE(submission_timestamp) = @submission_date
+    AND payload.processes.parent.scalars.devtools_accessibility_opened_count > 0
+)
+SELECT
+  submission_date,
+  normalized_channel,
+  COUNT(*) AS clients,
+  SUM(opened_count) AS panel_opens,
+  SUM(context_menu_opened_count) / (COUNT(*)) AS context_menu_opened_per_client_day,
```

Comment:
You may want to use `SAFE_DIVIDE` rather than `/` in case any of these counts returns 0.